nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/features/PlatformFunctions.java

```diff
@@ -178,7 +178,7 @@ public String getNextId(final String key, final String sequencerName) {
         return Long.toString(seq.getNextLong(key));
     }
 
-    public static String htmlEscape(String str) {
+    public String htmlEscape(String str) {
```

Comment:
This is not backward-compatible if someone was using the static method directly like @ThibArg shows in his NXP-27294 comment, so please add upgrade notes to the ticket explaining this.

Reply:
Yes will do ! Thanks !